### ISSUE TITLE
Gaussian-yolov3 will have false positives about small targets

### ISSUE THREAD

**Issue by Shame-fight** [AlexeyAB/darknet#4408]
Thanks for your work,I am your fans.
I have tried yolov3 and gauss_yolov3, 3 categories, one of which is a small target. More than 6000 images. Yolov3 was trained 18,000 times, the latter was trained 15,000 times. The result is that gauss will misidentify some objects in the background as a class (small targets), but this is basically not the case with yolov3. The loss of gauss_v3 is as high as 11561, but map@0.5 is more than 99. I don't know whether it is not enough training or other reasons.
Also,If you can, I would like to say: whether you can implement this version under ros, I really like its effect.but the weights replay to the darknet_ros package under preddie version, the effect will be greatly reduced. Thank you very much.

**Comment by yukai0406** [comment]
I have the same situation
Great loss...

**Comment by AlexeyAB** [comment]
This is strange that Gaussian has false-positives, since Gaussian is implemented to avoid false-positives by reducing probabilities.

Did you check your training dataset that there are no incorrect small boxes?

**Comment by HagegeR** [comment]
it happens for me too, my dataset is small with no errors at all, lots of small boxes FP

**Comment by AlexeyAB** [comment]
@yukai0406 @HagegeR 
Try to set and train
```
[Gaussian_yolo]
iou_normalizer=0.1
uc_normalizer=0.1
```
